### ISSUE TITLE
Add support for Pronto Ruby linter

### ISSUE THREAD

**Issue by alexmclean** [super-linter/super-linter#4850]
**Is your feature request related to a problem? Please describe.**
It appears that super-linter does not support linting only lines that have changed and has no plans to per this issue: https://github.com/super-linter/super-linter/issues/219

But this is something that we have really found useful working with a large legacy codebase that we can fix incrementally, and it is fully supported with Rubocop if you use Pronto: https://github.com/prontolabs/pronto

**Describe the solution you'd like**
Would you consider supporting Pronto within super-linter?  My understanding is that it can be used for running Rubocop or Flay, but Rubocop is the one that is already supported by super-linter.

**Comment by zkoppert** [comment]
Yeah, that sounds reasonable to me that we could support linting only diffed lines in in Ruby. I imagine that would need to be a configuration option though.

**Comment by ferrarimarco** [comment]
I had a quick look at Pronto. It seems that maintainers only released 2-3 releases over the last 2.5 years.

For now, we'll skip adding Pronto, but we can reconsider as it gains traction and adoption.
